### ISSUE TITLE
Context links in APM Traces

### ISSUE THREAD

**Issue by aenand** [DataDog/dd-trace-rb#2837]
**Is your feature request related to a problem? Please describe.**

After adding custom tags and spans to our APM traces we have a need to take a value from a particular trace to an external website to investigate more.

**Describe the goal of the feature**

Context links exist and are great in the dashboard view but would also be great if they were in APM traces where you could take values associated with a trace and link out to a website.

**Describe alternatives you've considered**

We could write the URL with query parameters into the tags but that seems like a lot of extra wasted space.

**How does `ddtrace` help you?**

We've used `ddtrace` to add Datadog APM instrumentation to our host for infrastructure data for a while but recently have explored adding business information into Datadog. We'd love to see Datadog get powerful enough for us to remove some of our other 3rd party dev tools and scale down our internal dev tools.

**Comment by TonyCTHsu** [comment]
👋 @aenand , thanks for reporting. I wasn't sure if I understand this correctly. 

> we have a need to take a value from a particular trace to an external website to investigate more.

> they were in APM traces where you could take values associated with a trace and link out to a website.

If you are creating custom spans with our manual instrument API, those values(external url? i guess?) could be tagged to the span directly. Could you provide me some examples?

**Comment by aenand** [comment]
@TonyCTHsu Yeah they could get tagged as an external URL but that adds a lot of overheard to the span data since we have to save the token + url whereas with [context links](https://docs.datadoghq.com/dashboards/guide/context-links/) in datadog we can dynamically generate the URL based on parameters.

**Comment by conner** [comment]
Just ran into this today - any chance this will be revisited or there's a creative workaround without tagging the spans with full URLs? We also have spans tagged with identifiers we could construct context links from.

**Comment by ivoanjo** [comment]
Hey folks! We don't (that I know of) have anything to announce yet regarding this feature (although I'd +1 it).

I do recommend asking Datadog support to raise a feature request for it -- https://www.datadoghq.com/support/ . Doing this allows us to tie the request to accounts, and to also get a view of interest across more customers (e.g. beyond just Ruby).
